### DISCUSSION TITLE
How to change a ui5-card background color ?

### DISCUSSION THREAD

**Discussion by BrunoSena97** [UI5/webcomponents#6266] [Q&A] [answered]
Hello,

I'm new to web components, and I'm using ui5 web components with vue js. I'm finding hard to customize the components, I'm using tailwind css on my project. 
As I understand, the UI Theme Designer is for SAP users only ?

Best regards and thank you for your work !

**Comment by ilhan007** [comment, marked as answer]
Hello @BrunoSena97 

You can read more on [Styling UI5 Web Components ](https://sap.github.io/ui5-webcomponents/main/playground/customizing/styles/) article.

The background of a `ui5-card` is defined via the `--sapTile_Background` global variable. You can try changing the `--sapTile_Background` variable in the scope of the `ui5-card`.

```css
ui5-card {
--sapTile_Background: red;
}
```
